patches/minecraft/net/minecraft/world/raid/Raid.java.patch

```diff
@@ -0,0 +1,26 @@
+--- a/net/minecraft/world/raid/Raid.java
++++ b/net/minecraft/world/raid/Raid.java
+       }
+    }
+ 
+-   static enum WaveMember {
++   public static enum WaveMember implements net.minecraftforge.common.IExtensibleEnum {
+       VINDICATOR(EntityType.field_200758_ax, new int[]{0, 0, 2, 0, 1, 4, 2, 5}),
+       EVOKER(EntityType.field_200806_t, new int[]{0, 0, 0, 0, 0, 1, 1, 2}),
+       PILLAGER(EntityType.field_220350_aJ, new int[]{0, 4, 3, 3, 4, 4, 4, 2}),
+@@ -783,5 +783,15 @@
+          this.field_221285_g = p_i50602_3_;
+          this.field_221286_h = p_i50602_4_;
+       }
++      
++      public static WaveMember create(String name, EntityType<? extends AbstractRaiderEntity> typeIn, int[] waveCountsIn) {
```

Comment:
Missing documentation.

Reply:
other classes that have this create method (eg: EntityClassification) dont have documentation either, in my opinion i dont see how this method would warrant documentation

Reply:
also if im not mistaken that method is already explained in the IExtensibleEnum class

Reply:
Yes that might be so, but the point i am trying to make is that the int[] is completely undocumented.
I personally do not care if it is on the create method or somewhere else.

But since you are touching this, adding a single line of documentation is what I ask for. It simply helps other modders out when they want to add a raid member themselfs.

Reply:
fair point